### DISCUSSION TITLE
Redirect

### DISCUSSION THREAD

**Discussion by notjohnless** [community/community#38013] [Other Feature Feedback, Questions, & Ideas]
Would it be possible to add a redirect from https://example.github.io/ to https://example.github.io/home? I want people to go to the home page when they first go to my site instead of the name and description of the site.

**Comment by radar-blip** [comment]
Most likely yes, but without seeing your code its hard to know how to solve.
Probably just a matter of pointing the link(s) in the right direction.

**Comment by jfullstackdev** [comment]
given the proper and valid link, you can redirect users anywhere
